eng/.docsettings.yml

```diff
@@ -48,6 +48,7 @@ known_content_issues:
   - ['sdk/synapse/azure-synapse-spark/README.md', '#4554']
   - ['sdk/synapse/azure-synapse-artifacts/README.md', '#4554']
   - ['sdk/synapse/azure-synapse-nspkg/README.md', '#4554']
+  - ['sdk/anomalydetector/azure-ai-anomalydetector/README.md', '#4554']
```

Comment:
FYI @mitchdenny added this in PR https://github.com/Azure/azure-sdk-for-python/pull/13141 at the bottom of the list so it is now in the list twice.